build.boot

```diff
@@ -8,6 +8,7 @@
     [tolitius/boot-check "0.1.4" :scope "test"]
 
     ;; Clojure/ClojureScript
+    ;; NB: Don't upgrade Clojure to 1.9 Alpha 15+ due to a Sente issue until it is resolved.
```

Comment:
Does this still apply? Seems no since the version here is 16. Any reason not to push it to 17?